package.json

```diff
@@ -63,17 +63,22 @@
     "@fortawesome/free-regular-svg-icons": "^6.4.2",
     "@fortawesome/free-solid-svg-icons": "^6.4.2",
     "@fortawesome/react-fontawesome": "^0.2.0",
+    "@langchain/core": "^0.2.18",
+    "@langchain/langgraph": "^0.0.31",
+    "@langchain/openai": "^0.2.5",
     "@types/lodash": "^4.14.195",
     "date-fns": "^2.28.0",
     "html-react-parser": "^5.0.7",
     "lodash": "^4.17.21",
     "markdown-it": "^13.0.1",
     "react-feather": "^2.0.9",
     "react-fontawesome-svg-icon": "^1.1.2",
-    "recharts": "^2.9.0"
+    "react-markdown": "^9.0.1",
```

Comment:
We already have a markdown component we're using here - I would see if you can use "markdown-it" as we ideally don't want to introduce a redundant dependency